src/components/PasswordDialog.vue

```diff
@@ -80,10 +77,10 @@ export default Vue.extend({
 			password: '',
 			showError: false,
 			dialogId: DIALOG_ID,
-			titleText: t('Authentication required'),
-			subtitleText: t('This action requires you to confirm your password'),
+			titleText: t('Confirm your password'),
+			subtitleText: t('This action needs authentication'),
 			passwordLabelText: t('Password'),
-			errorText: t('Failed to authenticate, please try again'),
+			errorText: t('Failed to authenticate, your password was wrong'),
```

Comment:
What about just `Wrong password` as it is an input description now?
Similar to the Login's page `Wrong username or password`, but without `username` :D